flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSource.java

```diff
@@ -77,6 +78,7 @@
 	private final RMQConnectionConfig rmqConnectionConfig;
 	protected final String queueName;
 	private final boolean usesCorrelationId;
+	protected RMQDeserializationSchema<OUT> deliveryDeserializer;
 	protected DeserializationSchema<OUT> schema;
```

Comment:
Elegant solution !!

Reply:
just a curious question "i am not that good with Java tbh ! 
Why a static function here ? why not a private method since it will only be used from within the class ?

Reply:
I think `static` is better just because there's no need to access any class fields or methods from within this. Probably could go either way, I'm no Java expert either!